app/models/security_log.rb

```diff
@@ -43,10 +108,50 @@ class SecurityLog < ApplicationRecord
   attribute :target_identifier, :string
   attribute :logged_at, :datetime
 
+  def self.log_event(action:, originator: nil, remote_ip: nil, target: nil, details: {})
+    details = details.merge({ ial_level: originator&.ial_level })
```

Comment:
on the Rails side, this value comes through as "ial2":
```
details: {"duration"=>386, "ial_level"=>"ial2"}
```
but on the Phoenix side it comes through as 2:
```
details: {"ial_level"=>2}
```
I guess the number is better for this purpose than the string, which can be gotten with `ial_level_for_database`:
```suggestion
    details = details.merge({ ial_level: originator&.ial_level_for_database })
```